app/apollo/resolvers/cluster.js

```diff
@@ -327,12 +355,21 @@ const clusterResolvers = {
 
         checkComplexity( queryFields );
 
+        let allAllowed = false;
+        try {
+          logger.info({req_id, user, org_id}, `${queryName} validating`);
```

Comment:
'validating' line should be right after `try {` line, should 'include' simple checks like checkComplexity.